tests/test_check_year.py

```diff
@@ -0,0 +1,38 @@
+"""
+Test check_year(y,s) 
+
+@author: unlu
+"""
+from ixmp.utils import check_year
+import pytest
+
+"Enter a string value for y to raise a ValueError. - in this case test should be successfull"
```

Comment:
The docstring ("Enter a string…") for a function should be.
- Enclosed in triple quotes `"""`.
- Inside the function.
- A full sentence ending in a period.